### ISSUE TITLE
Warn Card Czar if not all players have submitted answers before 'pick a winner'

### ISSUE THREAD

**Issue by FriendlyWP** [KevinBongart/cards_against_humanity#54]
Right now, if the Card Czar clicks 'pick a winner', any players who haven't submitted answers yet are prevented from doing so. While this might be on purpose, sometimes it's a mistake and there's no way to re-invite a player to submit an answer after the CC has clicked 'pick a winner'.  

A warning notice that 'not all players have submitted answers, are you sure you want to pick a winner?' would probably be enough to 'fix' it...? It just feels like a bug if you're not expecting it.

**Comment by KevinBongart** [comment]
Good idea and good candidate for an easy first contribution! Thanks @FriendlyWP
